packages/manager/modules/pci/src/projects/project/training/jobs/info/info.controller.js

```diff
@@ -38,11 +38,12 @@ export default class PciTrainingJobsInfoController {
 
     const catalog = this.getCatalogEntryF(this.job.spec.resources.flavor);
 
+    // Price per minute
```

Comment:
**(non-blocking):** Does the price can be retrieve via an API call instead?

Reply:
No you can't